chunkflow/chunk/base.py

```diff
@@ -378,17 +378,27 @@ def crop_margin(self, margin_size: tuple = None, output_bbox: Bbox=None):
             assert output_bbox is not None
             return self.cutout(output_bbox.to_slices())
     
-    def threshold(self, threshold: float):
-        seg = self > threshold
+    def threshold(self,
+                  threshold: float,
+                  remap_high: int = None,
+                  remap_low: int = None) -> np.ndarray:
+        # Threshold
+        seg = (self > threshold).astype(np.uint8)
+
+        # Optional remapping
+        iu8 = np.iinfo(np.uint8)
+        if remap_high is not None:
+            assert (remap_high >= iu8.min) and (remap_high <= iu8.max)
```

Comment:
Is it possible to write it as `assert iu8.min <= remap_high <= iu8.max`?

Reply:
Sure. I didn't know that's possible in Python.